### ISSUE TITLE
Lost sorting functionality after update to v1.1.1

### ISSUE THREAD

**Issue by daniefer** [Mimeo/VSTSExtension-ActivePullRequests#57]
I use to be able to sort each column by clicking on the header before the update to v1.1.1. This feature seems to have been broken or lost. I have been using this so I could see which open PRs need my attention or are waiting on my feedback again.

I did notice the new search bar could be a replacement for this, however it does not include  the "My Vote" column. I know #54 is currently open for fixing the wrong status going in that column, but once fixed I think this would be a nice replacement if the sorting was removed intentionally.

**Comment by joelbourbon** [comment]
The UI library was updated to use the one provided by Microsoft for Azure Devops.

The [table](https://developer.microsoft.com/en-us/azure-devops/components/table) supports sorting. It could be re-implemented.

**Comment by kyledodge** [comment]
I've gotten this re-implemented and working, once PR #61 is completed I will be able to submit another PR to add sorting to all columns

**Comment by joelbourbon** [comment]
@kyledodge I think we are ready for your 2nd PR 😉

**Comment by kyledodge** [comment]
It's up now ready for review.
